### PR TITLE
SP-558-action-history

### PR DESCRIPTION
**Pull request by jonhindle**

### CODE REVIEW

File: app/actionUpdate/get.controller.test.js [hunk 1]
```diff
@@ -96,6 +102,7 @@ describe('getActionUpdate', () => {
         })
       )
     })
+    it('should add a timeline entry if created and createdBy are defined', () => {})
```
Comment (Coupey) [inline app/actionUpdate/get.controller.test.js]: Missing test?

File: app/actionUpdate/get.controller.js [hunk 2]
```diff
@@ -25,11 +60,14 @@ const getActionUpdate = async (
       )
     )
     const backUrl = `${removeUrlLevels(path, 2)}`
-    const { intervention, description } = action
-    const actionText = intervention ? await getInterventionText(tokens, intervention) : description
-    const { motivationList } = await getMotivation(action, body, tokens).catch(error =>
-      catchAndReThrowError(`Could not retrieve motivation list`, error)
-    )
+    const { intervention, description, progress, created = '' } = action
+    const sausage = await Promise.all([
```
Comment (Coupey) [inline app/actionUpdate/get.controller.js]: Are we really accepting your sausage?

Comment (jonhindle) [reply app/actionUpdate/get.controller.js]: 🤦‍♂Coding while clearly unwell has its drawbacks...  
Removing now 👍

File: app/actionUpdate/get.controller.test.js (hunk 1 above)
Comment (jonhindle) [reply app/actionUpdate/get.controller.test.js]: 👍 I've put the test in, but it was sort of being tested anyway and will probably be redundant once the new version of the API is released.

**Pull request merged**